File: 2022/rust/src/bin/1a.rs
use aoc::input::read_stdin;
fn main() {
    let input_str = read_stdin();
    let calories: Vec<u32> = input_str
        .split("\n\n")
        .map(|elf| {
            elf.split("\n")
                .map(|line| line.parse::<u32>().unwrap())
                .sum::<u32>()
        })
        .collect();
    println!("{}", calories.iter().max().unwrap());
}

File: 2022/1a.js
const s = require("fs").readFileSync("inputs/day1.txt", "utf8");
const calories = s.split("\n\n").map((line) =>
  line
    .split("\n")
    .map((x) => parseInt(x))
    .reduce((acc, x) => acc + x, 0)
);

console.log(Math.max(...calories));
